app/code/Magento/AdminAnalytics/Test/Mftf/Test/AdminCheckAnalyticsTrackingTest.xml

```diff
@@ -22,14 +22,13 @@
             <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
                 <argument name="tags" value="config full_page"/>
             </actionGroup>
-            <reloadPage stepKey="pageReload"/>
         </before>
         <after>
             <magentoCLI command="config:set admin/usage/enabled 0" stepKey="disableAdminUsageTracking"/>
             <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
         </after>
 
-        <waitForPageLoad stepKey="waitForPageReloaded"/>
+        <actionGroup ref="ReloadPageActionGroup" stepKey="pageReload"/>
```

Comment:
It was written incorrect way ( 🤦🏻  ) so that your change is now backwards compatible.

To be backwards-compatible, you should keep old `stepKey` which is the extremely dirty way.
If we can introduce this in Magento 2.5 branch - then it would be not an issue.

Reply:
Hi @lbajsarowicz  
I have reverted changes for this file.
thanks